e2e/pages/maps/planting.py

```diff
@@ -0,0 +1,62 @@
+from playwright.sync_api import Page, expect, TimeoutError as PlaywrightTimeoutError
+from ..abstract_page import AbstractPage
+
+
+class MapPlantingPage(AbstractPage):
+    """The planting page of a map on permaplant"""
+    TITLE: str = 'PermaplanT'
+
+    def __init__(self, page: Page):
+        self.page = page
+        self.base_layer_radio = self.page.get_by_role("radio", name="select layer").first
+        self.plant_layer_radio = self.page.get_by_role("radio", name="select layer").nth(1)
+        self.plant_search_icon = self.page.locator(".flex-shrink > div > .flex > .inline-flex").first
```

Comment:
this doesn't look very stable, can you add some data-testid there?

Reply:
I see there is already many `data-tourid`s on elements I'm also testing. Do I just duplicate my testid's now or should I reuse those, any specific ideas about that?

Reply:
IMHO same as other ids, there might be some internal use. So I would continue with our general guideline: either user-facing text or `data-testid`. Please write this clearly in #653. But let Bushuo also review  #653 after you finalized it from your point of view.